include/class.topic.php

```diff
@@ -163,6 +168,20 @@ function getForms() {
         }
         return $this->_forms;
     }
+    
```

Comment:
To avoid multiple instances of organizations in `$this->_organizations`, I believe you should index the array, like this:

```
function getOrganizations() {
    if (!isset($this->_organizations)) {
        $this->_organizations = array();
        foreach ($this->organizations->select_related('organization') as $O) {
            $this->_organizations[$O->organization->id] = $O->organization;
        }
    }

    return $this->_organizations;
}
```